CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     logs, likewise use a pattern like the ES\_DISTRIBUTION\_TARGET\_PATTERN.
 
 ### Changed
+
+- **CUMULUS-2535**
+  - Ensured that the KPI cards would be updated on the executions tab whenever the page was updated
```

Comment:
Nit: can we change the wording here to indicate that this update is for all KPI cards?